apps/src/templates/instructions/CommitsAndReviewTab.jsx

```diff
@@ -0,0 +1,145 @@
+import React, {useState} from 'react';
+import PropTypes from 'prop-types';
+import {connect} from 'react-redux';
+import color from '@cdo/apps/util/color';
+import javalabMsg from '@cdo/javalab/locale';
+import Spinner from '@cdo/apps/code-studio/pd/components/spinner';
+import {ViewType} from '@cdo/apps/code-studio/viewAsRedux';
+import CodeReviewDataApiV2 from './codeReview/CodeReviewDataApiV2';
+import ReviewNavigator from './codeReview/ReviewNavigator';
+import Button from '@cdo/apps/templates/Button';
+
+export const VIEWING_CODE_REVIEW_URL_PARAM = 'viewingCodeReview';
+
+const CommitsAndReviewTab = ({
+  onLoadComplete,
+  channelId,
+  serverLevelId,
+  serverScriptId,
+  viewAsCodeReviewer,
+  viewAsTeacher,
+  codeReviewEnabled
```

Comment:
Is this tab shown if code reviews are not enabled?

Reply:
Currently the tab is always shown for javalab, you just can't navigate to a peers project if code review is turned off